python-test-samples/apigw-sqs-lambda-sqs/README.md

```diff
@@ -0,0 +1,155 @@
+
+[Top](#contents)
+
+
+```
+The SAM CLI installs dependencies defined in `src/requirements.txt`, creates a deployment package, and saves it in the `.aws-sam/build` folder. [Read the documentation](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-building.html).
+
+Use the following command to deploy your application package to AWS: 
+
+``` bash
+# deploy your application to the AWS cloud
+apigw-sqs-lambda-sqs$ sam build
+apigw-sqs-lambda-sqs$ sam deploy --guided
+```
+
+After running this command you will receive a series of prompts:
+
+* **Stack Name**: The name of the stack to deploy to CloudFormation. This should be unique to your account and region, and a good starting point would be something matching your project name. Use `apigw-sqs-lambda-sqs` as the stack name for this project. you'll need the stack name to run the Integration test.
+* **AWS Region**: The AWS region you want to deploy your app to.
+* **Confirm changes before deploy**: If set to yes, SAM CLI shows you any change sets for manual review before deployment. If set to no, the AWS SAM CLI will automatically deploy application changes.
+* **Allow SAM CLI IAM role creation**: Many AWS SAM templates, including this example, create AWS IAM roles required for the AWS Lambda function(s) included to access AWS services. By default, SAM CLI scopes these down to minimum required permissions. To deploy an AWS CloudFormation stack which creates or changes IAM roles, the `CAPABILITY_IAM` value for `capabilities` must be provided. If you don't provide permission through this prompt, you must explicitly pass `--capabilities CAPABILITY_IAM` to the `sam deploy` command.
+* **CheckOutputQueue may not have authorization defined, Is this okay?**:set to yes. since this is for testing only we can proceed without client autoriztion for API GW, but for production enviroment we do recommand to have it.
+
+* **Save arguments to samconfig.toml**: If set to yes, SAM CLI saves your choices to a configuration file inside the project, so that in the future you can just re-run `sam deploy` without parameters to deploy changes to your application.
+
+You can find your API Gateway Endpoint URL in the output values displayed after deployment. Take note of this URL for use in the logging section below. On subsequent deploys you can run `sam deploy` without the `--guided` flag. [Read the documentation](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-deploying.html).
+
+[Top](#contents)
+
+---
+
+## Run the Integration Test
+[test_api_gateway.py](tests/integration/test_api_gateway.py) 
+
+For integration tests, it is assumed the full stack is already deployed before testing. if not, you can use the following commands to deploy (see [Prerequisites](#prerequisites) section):
+```shell
+apigw-sqs-lambda-sqs$ sam build
+apigw-sqs-lambda-sqs$ sam deploy --guided
+```
```

Comment:
sam build/deploy redundant to the above - remove.

Reply:
fixed